frontend/src/metabase/entities/databases/forms.js

```diff
@@ -113,10 +113,22 @@ export const DEFAULT_SCHEDULES = {
   },
 };
 
+function concatURL(...urlParts) {
+  return urlParts.reduce(
+    (url, part) =>
+      typeof url !== "string" || typeof part !== "string"
+        ? undefined
+        : url + part.trim(),
+    "",
+  );
+}
+
```

Comment:
It looks like `concatURL` always takes two args. If so, maybe it could be simplified to remove `reduce`?

`function concatTrimmed(a, b) { return (a || "").trim() + (b || "").trim(); }`

That also drops the behavior that returns `undefined`. Do we rely on that?

Reply:
Good points -- no need to overdo it, so I'm using your function.